package.json

```
@@ -73,6 +73,7 @@
   },
   "homepage": "https://github.com/bbc/simorgh#readme",
   "dependencies": {
+    "@ampproject/toolbox-optimizer": "^2.8.3",
```

```suggestion
    "@ampproject/toolbox-optimizer": "2.8.3",
```
Our dependencies should be pinned renovate will raise a PR to correct this otherwise